lib/posix/fs.c

```diff
@@ -54,17 +57,43 @@ static inline void posix_fs_free_obj(struct posix_fs_desc *ptr)
 	ptr->used = false;
 }
 
+static int posix_mode_to_zephyr(int mf)
+{
+	int mode = mf & O_CREAT ? FS_O_CREATE : 0;
```

Comment:
Please use parentheses:

    int mode = (mf & O_CREAT) ? FS_O_CREATE : 0;

everywhere this idiom is used, to clarify intent and disambiguate from:

    int mode = mf & (O_CREAT ? FS_O_CREATE : 0);

which would be wrong.   It's not strictly necessary, but it is more clear, and it was done that one in one change already.

Reply:
Done